wiki/Tournaments/Official_support/en.md

```diff
@@ -181,7 +181,9 @@ We have historically had issues where unscrupulous organisers have geared tourna
 
 ### Can a player who has been eliminated from play in my tournament join the staff after the fact and not cause problems? {#player-join-staff}
 
-No. Participants who have been eliminated from the tournament/contest may not be enlisted as staff or assist in any organisational capacity **with the sole exception** of setting replays for mappool showcases. They can, however, be enlisted as a helper.
+No. Players who have been eliminated from the tournament/contest may not be enlisted as staff or assist in any organisational capacity with the sole exception of setting replays for mappool showcases. They can, however, be enlisted as a helper.
+
+A user who has passed screening but has not participated in any gameplay (including qualifiers) **AND** has not been scheduled for any matches does not qualify as a player/participant in our eyes, and therefore may be enlisted as staff (they have not been "eliminated" per se).
```

Comment:
if a non-participant hasn't played any matches, isn't it irrelevant (and not applicable) whether they have been eliminated?

if i understand this right, a user can participate as staff and later schedule matches to play in the tournament, but not later be part of staff once eliminated as a player, correct?

```suggestion
A user who has passed screening but has not participated in any gameplay (including qualifiers) **AND** has not been scheduled for any matches does not qualify as a player/participant in our eyes, and therefore may be enlisted as staff.
```

Reply:
```suggestion
A registrant who has passed screening but has not been scheduled for any matches **AND** has not participated in any gameplay (including qualifiers)matches is not yet a player of the tournament, and therefore may be enlisted as staff instead.
```

Reply:
```suggestion
A registrant who has passed screening but has not been scheduled for any matches **AND** has not participated in any gameplay (including qualifiers) is not yet considered a player/participant of the tournament, and may therefore still be enlisted as staff.
```